app/styles/_base.scss

```diff
@@ -38,3 +38,7 @@ img {
 input {
   border-radius: 0;
 }
+
+label, input, button {
+    outline: 0;
+}
```

Comment:
outlines make inputs accessible. instead of removing them entirely, may I suggest styling it to be something more appealing than the fuzzy blue line?

Reply:
Okay, so let's rollback it for button and inputs, but I will atleast remove it for basic `a` links ;)